cosmos/templates/footer.html

```diff
@@ -0,0 +1,55 @@
+{% load static %}
+<footer class="section-bg">
+    <div class="footer-top">
+        <div class="container pt-5 pb-3">
+            <div class="row">
+                <div class="col-md-4 text-center py-2">
+                    <h3>Cosmos</h3>
+                    <p>
+                        Den Dolech 2
+                        <br> MetaForum 0.090
+                        <br> TU Eindhoven
+                        <br> 5612 AZ Eindhoven
+                        <br> KvK number: 56429142
+                        <br>
+                        <br>
+                        <strong>Phone: </strong>+31 613 293 903
+                        <br>
+                        <strong>Email: </strong> cosmos@tue.nl
+                    </p>
+                </div>
+                <div class="col-md-4 d-none text-center d-md-block">
+                    <img src="{% static 'cosmos/img/cosmos_alternate_grey.svg' %}">
+                </div>
+                <div class="col-md-4 text-center py-2">
+                    <h4>Useful links</h4>
+                    <p>
+                        <a href="#">Privacy Policy</a> <br>
+                        <a href="#">Terms and Conditions</a> <br>
+                        <a href="#">Eindhoven Student Wiki</a> <br>
+                        <a href="#">Linktree</a> <br>
```

Comment:
add missing links pls

Reply:
We'll do this in another branch when we have a privacy policy page and T&C page